compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -722,10 +722,9 @@ setMatmulVectorDistributionConfig(IREE::GPU::TargetAttr target,
       targetSubgroupSize, pipelineConfig);
 }
 
-static LogicalResult
-setAttentionVectorDistributionConfig(IREE::GPU::TargetAttr target,
-                                     mlir::FunctionOpInterface entryPoint,
-                                     IREE::LinalgExt::AttentionOp op) {
+static LogicalResult setOnlineAttentionVectorDistributionConfig(
```

Comment:
I'd still call it setAttentionVectorDistributionConfig honestly. For me, both the operations are same.

Reply:
I d use the same logic (operations being the same) and be consistent with other setters in the file.

Maybe, I can do follow-up NFC to rename : 
set{YourFavouriteOp}VectorDistributionConfig -> setVectorDistributionConfig 

It feels redundant have it the function name when the argument type provides that information. I dont feel strongly either way. Do let me know.